config.go

```diff
@@ -226,12 +226,6 @@ func newQemuHypervisorConfig(h hypervisor) (vc.HypervisorConfig, error) {
 	kernelParams := h.kernelParams()
 	machineType := h.machineType()
 
-	for _, file := range []string{hypervisor, kernel, image} {
```

Comment:
presume you found this as the coverage said it was not covered, and you could not make it so as the upper layers stopped you getting this far :-)

Reply:
yes, `go test` discovered this was redundant code.